File: 프로그래머스/2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = []
    for i in range(len(prices)):
        tmp = 0
        for j in range(i+1, len(prices)):
            if prices[i] > prices[j]:
                tmp += 1
                break
            else:
                tmp += 1
        answer.append(tmp)
    return answer

File: 프로그래머스/2/42584. 주식가격/주식가격.js
function solution(prices) {
    const answer = [];
    for(let i = 0; i < prices.length; i++){
        let tmp = 0
        for(let j = i+1; j < prices.length; j++){
            if(prices[i] > prices[j]){
                tmp++
                break
            }else{
                tmp++
            }
        }
        answer.push(tmp)
    }
    return answer;
}